src/Handler/FlowErrorHandler.php

```diff
@@ -0,0 +1,165 @@
+     * @param int $level The minimum logging level at which this handler will be triggered
+     * @param bool $bubble Whether the messages that are handled can bubble up the stack or not
+     * @param int $maxColumnWidth The maximum column width that the message lines will have
+     */
+    public function __construct($to, $subject, $from, $level = Logger::ERROR, $bubble = true, $maxColumnWidth = 70)
+    {
+        parent::__construct($level, $bubble);
+        $this->to = is_array($to) ? $to : [$to];
+        $this->subject = $subject;
+        $this->addHeader(sprintf('From: %s', $from));
+        $this->maxColumnWidth = $maxColumnWidth;
+    }
+
+    /**
+     * Handles a record.
+     *
+     * All records may be passed to this method, and the handler should discard
+     * those that it does not want to handle.
+     *
+     * The return value of this function controls the bubbling process of the handler stack.
+     * Unless the bubbling is interrupted (by returning true), the Logger class will keep on
+     * calling further handlers in the stack with a given log record.
+     *
+     * @param array $record The record to handle
+     * @return bool true means that this handler handled the record, and that bubbling is not permitted.
+     *                        false means the record was either not processed or that this handler allows bubbling.
+     */
+    public function handle(array $record)
+    {
+        // TODO: Implement handle() method.
+        // Send an email
+
+        // Context
+        if (isset($record['context'])) {
+            if (isset($record['context']['exception'])) {
+                $exception = $record['context']['exception'];
+
+                if ($exception instanceof FlowException) {
+                    // Build up a semblance of a message
+                    $body = '------------' . "\n";
+                    $body .= 'An error was detected on your website:' . "\n";
+                    $body .= '------------' . "\n\n";
+                    $body .= 'Code: ' . $exception->getCode() . "\n";
+                    $body .= 'Message: ' . $exception->getMessage();
+
+                    $this->send($body);
+                }
+            }
+        }
+
+        return false;
+    }
+
+
+    /**
+     * Add headers to the message
+     *
+     * @param  string|array $headers Custom added headers
+     * @return self
+     */
+    public function addHeader($headers)
+    {
+        foreach ((array) $headers as $header) {
+            if (strpos($header, "\n") !== false || strpos($header, "\r") !== false) {
+                throw new InvalidArgumentException('Headers can not contain newline characters for security reasons');
+            }
+            $this->headers[] = $header;
+        }
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function send($content)
+    {
+        $content = wordwrap($content, $this->maxColumnWidth);
```

Comment:
This should be done with the SilverStripe `Email` class.

Don't forget to XML encode $content if sending it as HTML.

Reply:
I'll change it to the SS class. I based it off SilverStripe's NativeMailHandler, which just uses mail() weirdly.

And will do, on the XML encode.